### ISSUE TITLE
Flapping plugin max_retry TypeError

### ISSUE THREAD

**Issue by mind1m** [circus-tent/circus#951]
When I try to change flapping's max_retry option, I receive this exception :

```
$ circusd config/circus_flap.ini
2015-12-22 16:25:02 circus[7199] [INFO] Starting master on pid 7199
2015-12-22 16:25:02 circus[7199] [INFO] Arbiter now waiting for commands
2015-12-22 16:25:32 tornado.application[6043] [ERROR] Future <tornado.concurrent.Future object at 0x108400eb8> exception was never retrieved: Traceback (most recent call last):
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/gen.py", line 1014, in run
    yielded = self.gen.throw(*exc_info)
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/circus/arbiter.py", line 648, in manage_watchers
    yield list_to_yield
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/gen.py", line 1008, in run
    value = future.result()
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/concurrent.py", line 232, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 3, in raise_exc_info
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/gen.py", line 789, in callback
    result_list.append(f.result())
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/concurrent.py", line 232, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 3, in raise_exc_info
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/gen.py", line 1014, in run
    yielded = self.gen.throw(*exc_info)
  File "/Users/mind/sand/ecgate/env/lib/python3.5/types.py", line 179, in throw
    return self.__wrapped.throw(tp, *rest)
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/circus/watcher.py", line 535, in manage_processes
    yield self.spawn_processes()
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/gen.py", line 1008, in run
    value = future.result()
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/concurrent.py", line 232, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 3, in raise_exc_info
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/tornado/gen.py", line 282, in wrapper
    yielded = next(result)
  File "/Users/mind/sand/ecgate/env/lib/python3.5/types.py", line 199, in __next__
    return next(self.__wrapped)
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/circus/watcher.py", line 592, in spawn_processes
    res = self.spawn_process()
  File "/Users/mind/sand/ecgate/env/lib/python3.5/site-packages/circus/watcher.py", line 629, in spawn_process
    while nb_tries < self.max_retry or self.max_retry == -1:
TypeError: unorderable types: int() < str()
```

The config file I used is:

```
[circus]
check_delay = 5
endpoint = tcp://127.0.0.1:5555
pubsub_endpoint = tcp://127.0.0.1:5556

[plugin:flapping]
use = circus.plugins.flapping.Flapping
max_retry = 1

[watcher:command]
cmd = sleep
args = 2
```

**Comment by malaysarkar** [comment]
Any update on this?

**Comment by biozz** [comment]
I couldn't reproduce with provided config and it seems like this was fixed in a0723154f63a4877219ecd6ef92976a43c536064

**Comment by k4nar** [comment]
Yep, looks like this was fixed :+1: .
